mkdocs.yml

```diff
@@ -83,6 +83,7 @@ nav:
           - Dashboards: guides/web-ui/dashboards.md
           - Alerts (Beta): guides/web-ui/alerts.md
           - SQL Explorer: guides/web-ui/explore.md
+  - Understanding Charges: logfire-costs.md
```

Comment:
Suggested alternatives 

- Logfire Usage 
- Understanding your usage

That way the label is more agnostic to all user types (maybe the reference to Charges is scary for Free users?)